src/main/java/io/games/simple/webapp/configuration/WebSecurityConfiguration.java

```diff
@@ -20,7 +21,7 @@
     @Override
     protected void configure(HttpSecurity http) throws Exception{
         http.authorizeRequests()
-                .antMatchers("/h2-console/**", "/save", "/register").permitAll()
+                .antMatchers("/h2-console/**", "/save", "/register", "/tictactoe").permitAll()
```

Comment:
Czemu tictactoe jest w permitAll? Tam raczej dopiero po rejestracji będzie można wejść. Jak to tak tymczasowo sobie ustawiłeś dla swojej wygody to spoko, ale usuń to przed wrzuceniem na mastera, żeby nie zapomnieć później i nie szukać błędu.

Reply:
Fixed adn ready to merge